helm-charts/seldon-core-operator/templates/clusterrole_seldon-manager-role.yaml

```diff
@@ -91,9 +91,21 @@ rules:
   - update
   - watch
 - apiGroups:
-  - keda.sh/v1alpha1
+  - keda.sh
```

Comment:
Why did you remove v1alpha1 in the logs it seems to be saying
```
E1021 11:49:08.797354       1 reflector.go:178] pkg/mod/k8s.io/client-go@v0.18.8/tools/cache/reflector.go:125: Failed to list *v1alpha1.ScaledObject: scaledobjects.keda.sh is forbidden: User "system:serviceaccount:seldon-system:seldon-manager" cannot list resource "scaledobjects" in API group "keda.sh" at the cluster scope
```

Reply:
I think in RBAC, apiGroups shouldn't include the version, e.g. https://github.com/SeldonIO/seldon-core/blob/master/helm-charts/seldon-core-operator/templates/clusterrole_seldon-manager-role.yaml#L102

https://github.com/kedacore/keda/blob/v2/config/rbac/role.yaml#L53